bitnami/harbor/templates/trivy/trivy-sts.yaml

```diff
@@ -0,0 +1,182 @@
+      securityContext: {{- include "common.tplvalues.render" (dict "value" .Values.podSecurityContext "context" $) | nindent 8 }}
+      {{- end }}
+       */}}
+      automountServiceAccountToken: {{ .Values.trivy.automountServiceAccountToken }}
+      initContainers:
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+        - name: volume-permissions
+          image: {{ include "harbor.volumePermissions.image" . }}
+          imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+          command:
+            - sh
+            - -c
+            - |
+              mkdir -p "{{ .Values.trivy.cacheDir }}"
+              mkdir -p "{{ .Values.trivy.cacheDir }}/trivy"
+              mkdir -p "{{ .Values.trivy.cacheDir }}/reports"
+              chown -R "{{ .Values.podSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "{{ .Values.trivy.cacheDir }}"
+          securityContext:
+            runAsUser: 0
+          {{- if .Values.volumePermissions.resources }}
+          resources: {{- toYaml .Values.volumePermissions.resources | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - name: job-logs
+              mountPath: /var/log/jobs
+              subPath: {{ .Values.persistence.persistentVolumeClaim.jobservice.subPath }}
+      {{- end }}
+      {{- if .Values.trivy.initContainers }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.trivy.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: trivy
+          image: {{ include "harbor.trivyImage" . }}
+          imagePullPolicy: {{ .Values.trivyImage.pullPolicy | quote }}
+          {{/* TODO restore the proper security context
+          {{- if .Values.containerSecurityContext }}
+          securityContext: {{- include "common.tplvalues.render" (dict "value" .Values.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+           */}}
+          {{- if .Values.trivy.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.trivy.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.trivy.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.trivy.args "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.trivyImage.debug | quote }}
+            {{- if .Values.trivy.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.trivy.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            - configMapRef:
+                name: {{ template "harbor.trivy" . }}-envvars
+            - secretRef:
+                name: {{ template "harbor.trivy" . }}-envvars
+            {{- if .Values.trivy.extraEnvVarsCM }}
+            - configMapRef:
```

Comment:
do not we need checksums for `extraEnvVarsCM` and `extraEnvVarsSecret`?

Reply:
We don't render them